dune

```diff
@@ -0,0 +1,4 @@
+(library
+ (name PRNG)
+ (public_name pringo)
+ (libraries))
```

Comment:
The `libraries` stanza can be removed here.

Reply:
Done!